processing-tests/common/test/kotlin/com/livefront/sealedenum/compilation/hierarchy/SealedClassHierarchy.kt

```diff
@@ -229,25 +229,25 @@ public fun FirstClassHierarchy.A.B.Companion.valueOf(name: String): FirstClassHi
 
 class SecondClassHierarchy {
 
-    sealed class A {
+    sealed class Z {
```

Comment:
Random, funny thing I noticed here: I setup most of these tests where alphabetical order was the same as source order. That ended up causing less test failures upon updating to Kotlin 1.7.0 then I was expecting, just due to how I named these objects.

Here, I flipped the order entirely to ensure the change in logic works.